REVIEW.md

```diff
@@ -0,0 +1,219 @@
+
+**Note**: The difference between an objective improvement and a subjective preference isn’t always clear.
+Reviewers should recall that code review is primarily about reducing risk in the project.
+When reviewing code, one should aim at preventing situations which may require a bug fix, a deprecation, or a retraction.
+
+- [ ] Improvements
+    - Could the code easily be rewritten to run much more efficiently for relevant settings?
+    - Is the code backwards compatible with previous versions?
+      (Or is a deprecation cycle necessary?)
+    - Is there feedback that would be helpful for solving future issues surrounding this change?
+      Are there non-urgent concerns that the team should be made aware of?
+    - If a bug is fixed, does the PR include a non-regression test?
+    - Are there images that should be added?
+      Are there images that should be resized or optimized in another way?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns in Spyder?
+      If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+
+## For documentation-focused review
+
+### Required review
+
+The following are areas of review the community expects to be covered in the review of a documentation-focused (sometimes abbreviated as docs) PR.
+In general, reviewers need to consider whether the content is a good fit, accurate and generally clear to readers, and builds and renders with no simmediate user-visible issues.
+
+**Motivation**
+
+- [ ] Does the change improve the documentation?
+- [ ] Would it help multiple users to have this as a part of the documentation?
+- [ ] Is it appropriate for the documentation and the section it is placed in?
+
+**Content**
+
+- [ ] Is the information accurate?
+- [ ] Is the change complete, such that it won’t confuse users if it were to appear in the documentation as-is?
+- [ ] Is it clear what the text and images are trying to communicate?
+
+**Images/GIFs**
+
+- [ ] Are screenshots/GIFs reasonably legible and appropriate for the topic?
+- [ ] Are all images/GIFs scaled and optimized?
+If needed, reviewers should offer to help with this.
+
+**Technical**
+
+- [ ] Do the tests pass in the continuous integration build?
+If appropriate, help the contributor understand why tests failed.
+- [ ] Does any added/changed content render correctly, without user-visible issues?
+- [ ] Is the content free from typos and unambiguous grammar errors (review the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md) for more information)?
```

Comment:
I don't think `unambiguous grammar errors`are quite as unambiguous to many people, so I was trying to provide literally any context for the statement. If you have another link that describes what kind of errors this includes, that would be great. 

I am a native english speaker (lowercase e is intentional), and I'm not certain what kind of feedback this statement entitles reviewers to, so I think there's a chance it's unclear to others as well. However, I don't want this PR to stay in review for much longer, so if there is no other resource I can restore the unlinked version.

Reply:
Basically, if English were in a programming language, syntax (grammar) that would produce an error (i.e not following the grammar, capitalization, spelling and punctuation rules of the language, e.g. "This is a comma splice, this is a comma splice too"), or semantics (meaning) that is clearly ambiguous (see [examples](https://en.wikipedia.org/wiki/List_of_linguistic_example_sentences#Ambiguity)) or not the intended behavior (i.e. saying something different from what you mean, e.g. accidentally writing "Spyder isn't a great IDE" if the opposite was meant). This contrasts with issues of diction, phrasing, verbosity, repetition, etc that make the text harder to read and understand but not unambiguously incorrect (in language or meaning), style conventions, consistency issues, awkward or unidiomatic but not incorrect or clearly ambiguous syntax, formatting matters, or other items enumerated and described in the section below.

For the purpose of these docs, the AP style guide serves as the final authoritative reference for these matters (as mentioned in the style guide). However, while essentially all of its important rules can be found scattered around a large variety of unofficial third party online resources (and practically speaking, Googling it is the approach someone would take if they weren't sure of something—even myself, I suppose, if that ever were to become necessary, rather than digging out my old copy of AP), like all the common style guides access to guide itself requires a subscription, printed copy, etc, so there's no one place to link to.

In practice (ultimately, like almost any of these points if you drill down far enough), this comes down to an reviewer's best judgement in good faith on the basis of their knowledge, expertise and experience that something is a syntax error, lexical ambiguity or semantic mistake. In the extraordinary case that a reviewer was acting in bad faith and engaged in a large-scale, systemic, purposeful effort to claim style issues were grammar errors, other reviewer(s) with relevant expertise in this area would inherently be able to pick up on it, and cross-referencing a representative sample of cases with third party resources would serve to confirm it. But this sort of malfeasance would be wholly unprecedented in the history of the project and completely out of character of anyone I'd hope we would consider accepting as a member.

> I am a native english speaker (lowercase e is intentional)

Could you explain a little more what you mean to imply here? I'm afraid I don't quite grasp it, sorry. At least by its own capitalization rules, as a proper noun, English is always capitalized. Is there an emerging, not-yet-standardized usage here that I'm yet unaware of? Or is this intended to make a point that I'm not picking up on?

Reply:
With all of that said (at perhaps too great length), perhaps there is something you can distill out of that which would be helpful here to alleviate your concerns?